Animations/Fade/Down.js

```diff
@@ -26,5 +28,5 @@ var FadeDown = {
     'transform': 'translateY(-20px)'
   }
 };
-//Use makeTransition function from main Tux Animation module to create wrapping animation componenet and pass in the default params
-module.exports = Animation.makeTransition('FadeDown', FadeDown);
+//Use makeAnimation function from main Tux Animation module to create wrapping animation componenet and pass in the default params
+module.exports = makeAnimation('FadeDown', FadeDown);
```

Comment:
This does not need to have `'FadeDown'` passed into it since we have defined a `className` property in `FadeDown`.